### DISCUSSION TITLE
How "Automatically synchronise time" works?

### DISCUSSION THREAD

**Discussion by Jesus-M** [JurajNyiri/HomeAssistant-Tapo-Control#779] [General]
Hello,

I am using **HomeAssistant** and the **Tapo: Cameras Control** since recently, and today I noticed all my Tapo cameras had the time wrong (+1 hour). I double and triple checked the cameras configuration, the time zone, and NTP server time correctness, and all seemed to be fine.

Then I remembered the option `Automatically synchronise time` in the integration that I had enabled for all cameras. I disables for one of the cameras, rebooted it and ... problem solved.

So, I am guessing the source for the `Automatically synchronise time` is likely wrong (or not considering time zones), I don't know, the thing is I'd like to understand what this option source is for the time to understand why it adds one hour to the local time.

Thanks

**Comment by JurajNyiri** [comment]
Please continue in https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/307
